qiskit/test/mock/fake_backend.py

```diff
@@ -38,6 +62,257 @@ def __init__(self, token="123456", url="https://"):
         self.project = "project"
 
 
+class FakeBackendV2(BackendV2):
+    """This is a dummy bakend just for resting purposes. the FakeBackendV2 builds on top of the BackendV2 base class."""
+
+    dirname = None
+    conf_filename = None
+    props_filename = None
+    defs_filename = None
+    backend_name = None
+
+    def __init__(self):
+        self._conf_dict = self._get_conf_dict_from_json()
+        self._props_dict = self._set_props_dict_from_json()
+        self._defs_dict = self._set_defs_dict_from_json()
+        super().__init__(
+            provider=None,
+            name=self._conf_dict.get("backend_name"),
+            description=self._conf_dict.get("description"),
+            online_date=self._conf_dict.get("online_date"),
+            backend_version=self._conf_dict.get("backend_version"),
+        )
+        self._target = None
+        self._qubit_properties = None
```

Comment:
Same with this, is there a reason to just not populate this on init?

Reply:
Same as above. I will modify it.